components/GlobalStyles.js

```diff
@@ -0,0 +1,30 @@
+import { createGlobalStyle } from 'styled-components';
+
+export const GlobalStyle = createGlobalStyle`
+  * {
+    font-family: 'Open Sans', sans-serif;;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+`;
+
+export const theme = {
+  colors: {
```

Comment:
As I've pointed out earlier, perhaps consider names such as "background", "accent", "notification", "text", etc. I like primary and secondary, as long as these colors are proportionally equivalent to their name. So primary would be the color that is used the most, etc.